data/json/vitamin.json

```diff
@@ -83,6 +83,19 @@
     "rate": "100 s",
     "disease_excess": [ [ 10, 216 ] ]
   },
+  {
+    "id": "vit_mme",
+    "type": "vitamin",
+    "vit_type": "drug",
+    "name": { "str": "Morphine Milligram Equivalent (percentile)" },
+    "//": "1 vit_mme is equivalent to 1% of an MME.  Need to adjust excess and rate at minimum.",
+    "excess": "opiate_eff",
+    "min": 0,
+    "max": 40000,
+    "//": "Takes 30 min to kick in, lasts 6 hours",
```

Comment:
For multiple comments in one object I believe you have to do `"//"`, `"//1"`, `"//2"`, etc.

Reply:
yeah but that's a copy from error.